contracts/core/LensHub.sol

```diff
@@ -875,6 +917,23 @@ contract LensHub is ILensHub, LensNFTBase, VersionedInitializable, LensMultiStat
         );
     }
 
+    function _setDefaultProfile(uint256 profileId, address wallet) internal {
+        // you should only be able to map this to the owner OR dead address
+        if (wallet != address(0)) {
+            _validateCallerIsProfileOwner(profileId, wallet);
```

Comment:
think you should do this validation in both cases
would be nice to add a test case